cfme/tests/openstack/infrastructure/test_hosts.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from utils import testgen
+from cfme.web_ui import Quadicon, toolbar
+from cfme.infrastructure.host import Host
+from cfme.infrastructure.provider.openstack_infra import OpenstackInfraProvider
+from cfme.web_ui import InfoBlock
+from cfme.fixtures import pytest_selenium as sel
+from cfme.configure.tasks import is_host_analysis_finished
+from utils.wait import wait_for
+
+
+pytest_generate_tests = testgen.generate(testgen.providers_by_class,
+                                         [OpenstackInfraProvider],
+                                         scope='module')
+pytestmark = [pytest.mark.usefixtures("setup_provider_modscope")]
+
+
+def test_host_configuration(provider, soft_assert):
+    provider.load_details()
```

Comment:
Please use `navigate_to(provider, 'Details')` here, we plan to remove load_details() class methods.